config/helm/webhook-watchfilter.yaml

```diff
@@ -8,6 +8,10 @@ webhooks:
     objectSelector:
       matchLabels:
         cluster.x-k8s.io/watch-filter: capi
+  - name: default.azureclustertemplate.infrastructure.cluster.x-k8s.io
+    objectSelector:
+      matchLabels:
+        cluster.x-k8s.io/watch-filter: capi
```

Comment:
We might need to rethink if we want to have watch-filter label selector added here conditionally in helm, so we include them only in vintage, as CAPZ clusters will not need them.

Reply:
yes, this caused us issues with the first tests on glippy and indeed in that branch it has been removed https://github.com/giantswarm/cluster-api-provider-azure-app/pull/69/files :+1: